app/viam_client.go

```diff
@@ -67,6 +70,23 @@ func CreateViamClientWithAPIKey(
 	return CreateViamClientWithOptions(ctx, options, logger)
 }
 
+// CreateViamClientFromEnvVars creates a ViamClient from env vars set by the module manager.
+// It is intended to be used from within modular resources, and will likely fail if called
+// in another context.
```

Comment:
Can you update this comment as well to enumerate which env vars are need to be set? while i think 95% of the time this will be done via modules, i can see sometimes this actually just being used by normal users too